packages/neos-ui/src/Containers/LeftSideBar/style.css

```diff
@@ -15,10 +15,18 @@
     &:after {
         clear: both;
     }
+
+    @media(max-width: 540px) {
```

Comment:
Ideally we try to extract all constants, I think it definitely makese sense to do the same for breakpoints as well, so it would be easier to manage them.

Reply:
Yes, it does. I did not manage to get the constants working inside the media query – is there a trick?

Reply:
Since we are using postcss, we could use this package: https://www.npmjs.com/package/postcss-media-variables

Reply:
Does not go well with our PostCSS-configuration sadly ... https://github.com/WolfgangKluge/postcss-media-variables/issues/8

Reply:
Ah, too bad